client/src/processing.rs

```diff
@@ -43,6 +43,21 @@ impl PostProcBin {
     }
 }
 
+pub fn streamline_pixel_color(pixels: &Vec<u8>, post_proc: &PostProcBin) -> Vec<u8> {
+    if display::device::CURRENT_DEVICE.color_samples() == 1 {
+        pixels
+            .chunks(4)
+            .map(|chunks| {
+                let rgb =display::color::Color::from_rgb(chunks);
+                rgb.gray()
```

Comment:
fixes #39 
we now consider all 3 color channels ... and relay upon the Plato's to color (luminance)